src/mappings/UController.graphql

```diff
@@ -0,0 +1,63 @@
+type UControllerOrganization @entity {
+    avatarAddress: ID!
```

Comment:
how about 'avatarAddress : Avatar! '?

Reply:
Why do we need another entity? also Avatar is another contract.

Reply:
this is the power of Graph(ql).
controller point to Avatar

Reply:
But we have no Avatar entity right now

Reply:
ok. lets do it only for rep